### ISSUE TITLE
No Man's Sky on Arch

### ISSUE THREAD

**Issue by FlanGrande** [FakeMichau/fgmod#3]
Hi there,

I've been trying to get this to work on No Man's Sky on Arch to no avail.
I'm not sure what info would be useful or if you can even do something about it. My guess is that it just simply doesn't work?
I have a Nvidia 3060 and I installed the game through Steam. I am using fgmod in the launch options as advised by the tool.

Am I supposed to be forcing a specific version of Proton?

**Comment by FakeMichau** [comment]
NMS is currently the only Vulkan game that uses Streamline so there are some growing pains. Nukem made FSR FG somewhat work on Nvidia but DLSS Enabler, especially on AMD, has even more issues. 

Can you share dlss-enabler.log and dlssg_to_fsr3.log? Just zip them

**Comment by FlanGrande** [comment]
Where would they be created? I've checked both fgmod and NMS's Binaries folder and I can't find them there.

**Comment by FakeMichau** [comment]
Yes it should be in the Binaries folder. Could you do two thing:
- confirm that dxgi.dll and dlss-enabler.dll are actually in this folder
- remove the SETTINGS/graphicsettings.mxml file, a member of the discord server that uses nvidia always had to delete that file

**Comment by FlanGrande** [comment]
Both of the .dll files are there.
I tried removing that file, then launching the game again, and the behaviour was the same. No log files have been created either. This is most likely normal, but just in case, the file you asked me to delete gets recreated.

Sorry to ask again, but, should I be using Proton? If so, is any version favoured? I've tried both with and without with no perceptible difference.

In case it helps, here's a screenshot of the Binaries folder:
![image](https://github.com/user-attachments/assets/c7779686-8406-41bb-9e61-5ec05f71e47a)

Tomorrow I will try and see if I can find something on Steam/NMS logs.

**Comment by FakeMichau** [comment]
I use Proton Experimental Bleeding Edge. It would seem as if streamline doesn't load dxgi.dll for you.
Try following steps:
1. Remove ``/home/user/fgmod/fgmod`` from the launch options
2. Add ``WINEDLLOVERRIDES=winmm=n,b`` to the launch options
3. Remove ``dxgi.dll`` from the game folder
4. Rename ``dlss-enabler.dll`` to ``winmm.dll``

And launch the game again

**Comment by FlanGrande** [comment]
[logs.tar.gz](https://github.com/user-attachments/files/16841095/logs.tar.gz)

That did the trick, finally manage to get logs going.
When I opened the game, it initially seemed to run, it showed the Hello Games logo, then the visuals froze, but I could here what I can only assume was the main menu through my headphones.

**Comment by FakeMichau** [comment]
So you probably have the exact same issue as me. If you use a shortcut to make the game exist fullscreen it should run again for a few frame and freeze and just repeat it.

It's not in a good state but I also don't fully understand what's going on here. I just make a hopefully useful installer :)

**Comment by FlanGrande** [comment]
It's alright. At least the game runs fine without it anyways.

**Comment by slynobody** [comment]
similar problem currently with amd-card on deb.

**Comment by FakeMichau** [comment]
> also wd legions does currently _not_ work

It will be best to create a separate issue for that. Please add all log files from the game folder into it when you do. 


> Do you plan to include both in the future?

NMS is mostly broken in the mod that my installer installs so I'm at the mercy of that mod. Tho I can tell you that there were many attempts to fixing it, both by [Nukem](https://github.com/Nukem9) and by [nitec](https://github.com/cdozdil) but after many failures the motivation to making Vulkan Streamline work is quite low.
